guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -42,40 +43,42 @@ endif::[]
 A {SmartProxy} behind a load balancer takes precedence over a {SmartProxy} selected in the {ProjectWebUI} as the content source of the host.
+* On the *General* tab, in the *Download utility* field, you can select `wget` to register your host using a `wget` command.
+By default, {Project} generates a `curl` command.
 * On the *General* tab, you can select the *Insecure* option to make the first call insecure.
-During this first call, the host downloads the CA file from {Project}.
-The host will use this CA file to connect to {Project} with all future calls making them secure.
+During this first call, your host downloads the CA file from {Project}.
+Your host will use this CA file to connect to {Project} with all future calls making them secure.
 +
 {Team} recommends that you avoid insecure calls.
 +
-If an attacker, located in the network between {Project} and a host, fetches the CA file from the first insecure call, the attacker will be able to access the content of the API calls to and from the registered host and the JSON Web Tokens (JWT).
-Therefore, if you have chosen to deploy SSH keys during registration, the attacker will be able to access the host using the SSH key.
+If an attacker, located in the network between {Project} and your host, fetches the CA file from the first insecure call, the attacker will be able to access the content of the API calls to and from your host and the JSON Web Tokens (JWT).
+Therefore, if you have chosen to deploy SSH keys during registration, the attacker will be able to access your host using the SSH key.
 * On the *Advanced* tab, in the *Repositories* field, you can list repositories to be added before the registration is performed.
 You do not have to specify repositories if you provide them in an activation key.
 * On the *Advanced* tab, in the *Token lifetime (hours)* field, you can change the validity duration of the JSON Web Token (JWT) that {Project} uses for authentication.
-The duration of this token defines how long the generated `curl` command works.
+The duration of this token defines how long the generated registration command works.
 +
-Note that {Project} applies the permissions of the user who generates the `curl` command to authorization of hosts.
+Note that {Project} applies the permissions of the user who generates the registration command to authorization of your host.
 If the user loses or gains additional permissions, the permissions of the JWT change too.
 Therefore, do not delete, block, or change permissions of the user during the token duration.
 +
 The scope of the JWTs is limited to the registration endpoints only and cannot be used anywhere else.
 
 .CLI procedure
-. Use the `hammer host-registration generate-command` to generate the `curl` command to register the host.
-. On the host that you want to register, run the `curl` command as `root`.
+. Use the `hammer host-registration generate-command` to generate the `curl` command to register your host.
```

Comment:
What about CLI? Is this working for hammer as well? Or is wget planned to be implemented later?

Reply:
Just found out myself: 
![image](https://github.com/user-attachments/assets/955c7ffd-b548-4f0d-9353-76963ad98b7f)

And I tried it, it actually does generate a command. So please add this to the documentation.

Reply:
I don't think it's necessary to mention the option specifically. They can see it in Hammer help as said below.

Reply:
The sentence above only mentions the curl command but not wget at all. That is what I stumbled across.

Reply:
Oh, I see! In that case I suggest:
```suggestion
. Use the `hammer host-registration generate-command` to generate the registration command to register your host.
```

Reply:
I applied the suggested changes for the CLI procedure. The API procedure only refers to the API reference. The `download_utility` is documented there.

Regarding the Ansible procedure: AFAIK for now, the usage of `download_utility` has to be documented in [plugins/modules/registration_command.py in foreman-ansible-modules](https://github.com/theforeman/foreman-ansible-modules/blob/develop/plugins/modules/registration_command.py). But I'll have to take another look and would open a PR accordingly.